Rakefile

```diff
@@ -246,16 +246,19 @@ Groonga #{version} (#{latest_release_date}) has been released!
 #{latest_release_note_summary}
 See: #{latest_release_url}#{latest_release_anchor}
     ANNOUNCE
+    puts latest_release_announce
 
-    x_credentials = {
-      api_key: ENV["X_API_KEY"],
-      api_key_secret: ENV["X_API_KEY_SECRET"],
-      access_token: ENV["X_ACCESS_TOKEN"],
-      access_token_secret: ENV["X_ACCESS_TOKEN_SECRET"]
-    }
-    x_client = X::Client.new(**x_credentials)
-    tweet_body = { text: latest_release_announce }
-    x_client.post("tweets", tweet_body.to_json)
+    unless ENV["DRY_RUN"]
+      x_credentials = {
+        api_key: ENV["X_API_KEY"],
+        api_key_secret: ENV["X_API_KEY_SECRET"],
+        access_token: ENV["X_ACCESS_TOKEN"],
+        access_token_secret: ENV["X_ACCESS_TOKEN_SECRET"]
+      }
+      x_client = X::Client.new(**x_credentials)
+      tweet_body = { text: latest_release_announce }
```

Comment:
Can we extract this out of `unless ENV["DRY_RUN"]`?
We should execute our codes as much as possible to detect problems before we release a new version.

Reply:
fix: [cd07398](https://github.com/groonga/groonga/pull/2324/commits/cd0739858247ef61a2d7c63e42d22a200a67fd4e) Sure. I run the codes as much as we can for testing.